src/Management/src/Endpoint/Trace/ITraceEndpoint.cs

```diff
@@ -4,6 +4,6 @@
 
 namespace Steeltoe.Management.Endpoint.Trace;
 
-public interface ITraceEndpoint : IEndpoint<List<TraceResult>>
+public interface ITraceEndpoint : IEndpoint<IList<TraceResult>>
```

Comment:
Update: we agreed to prefer TArgument/TResult.

Reply:
Done in the Separate_middlewareoptions PR

Reply:
It looks like we settled on `IEndpointHandler<in TArgument, TResult>`, where endpoint-specific interfaces use basic types when possible. Otherwise, they use types named {endpoint-name}Request and {endpoint-name}Response.

There are a few exceptions to the above convention, though:
- `IEnvironmentEndpointHandler` uses `EnvironmentDescriptor` for `TResult`. The type isn't used anywhere outside this handler, so I suspect it should be renamed.
- `IMappingsEndpointHandler` uses `ApplicationMappings` for `TResult`. The type isn't used anywhere outside this handler, so I suspect it should be renamed.

Reply:
It isn't much of a convention though. What in this case is the difference between the a system type and a steeltoe defined type, when the types are descriptive. In this case an environmentDescriptor vs Env Endpoint. Also, It's the Env endpoint not the Environment endpoint. How is EnvironmentResponse more descriptive than the EnvironmentDescriptor. These changes seem arbitrary and unnecessary.

Reply:
I changed the return types to reflect the discussion.